prac_09/sort_files.py

```diff
@@ -0,0 +1,41 @@
+"""
+CP1404/CP5632 Practical
+Program to sort files into directories based on file type
+By Christopher Caferra
+"""
+
+import os
+import shutil
+
+
+def main():
+    """Sort files into directories based on file extension"""
+    extensions = []
+    extension = ''
+    os.chdir('FilesToSort')
+    for filename in os.listdir('.'):
+        file_type = filename[filename.find('.') + 1:]
```

Comment:
Could also change to `os.path.basename(filename)` since it may make your intentions a little more clear (that you want the entension of a file) and therefore the code more readable